### PR TITLE
little fix

### PR DESCRIPTION
**Pull request by andriy-shymkiv**
- [DEMO LINK](https://andriy-shymkiv.github.io/layout_catalog_grid/)

### CODE REVIEW

File: src/styles/blocks/_catalog.scss [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+    grid-template-columns: $cardWidth;
+  }
+
+  &__card {
+    display: grid;
+    justify-content: center;
+    align-content: center;
+    grid-template-columns: auto;
+
+    margin-bottom: 48px;
```
Comment (taraselnik) [inline src/styles/blocks/_catalog.scss:27]: why not 'gap' above?

**Review by taraselnik — APPROVED** [review]
enjoy

File: src/styles/blocks/_catalog.scss (hunk 1 above)
Comment (illiarerun) [reply src/styles/blocks/_catalog.scss:27]: agree, use gap for it
```suggestion
```

File: src/styles/blocks/_catalog.scss [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+
+    width: $cardWidth;
```
Comment (illiarerun) [inline src/styles/blocks/_catalog.scss:29]: width here is not needed it should be controlled by grid
```suggestion
```

File: src/styles/blocks/_catalog.scss [hunk 3]
```diff
@@ -0,0 +1,35 @@
+.catalog {
+  display: grid;
+  justify-content: center;
+  grid-template-columns: repeat(4, $cardWidth);
+  grid-column-gap: 48px;
+
+  margin: 0 auto;
```
Comment (illiarerun) [inline src/styles/blocks/_catalog.scss:7]: use grid options to center items
```suggestion
```

File: src/styles/blocks/_catalog.scss [hunk 4]
```diff
@@ -0,0 +1,35 @@
...
+    grid-template-columns: repeat(3, $cardWidth);
+  }
+
+  @media (max-width: 768px) {
+    grid-template-columns: repeat(2, $cardWidth);
+  }
+
+  @media (max-width: 488px) {
+    grid-template-columns: $cardWidth;
+  }
```
Comment (illiarerun) [inline src/styles/blocks/_catalog.scss:19]: There should be two columns **starting** at 488px
image

File: src/styles/blocks/_catalog.scss [hunk 5]
```diff
@@ -0,0 +1,35 @@
...
+
+  margin: 0 auto;
+
+  @media (max-width: 1024px) {
+    grid-template-columns: repeat(3, $cardWidth);
+  }
+
+  @media (max-width: 768px) {
+    grid-template-columns: repeat(2, $cardWidth);
+  }
```
Comment (illiarerun) [inline src/styles/blocks/_catalog.scss:15]: same here

File: src/styles/blocks/_catalog.scss [hunk 6]
```diff
@@ -0,0 +1,35 @@
...
+  display: grid;
+  justify-content: center;
+  grid-template-columns: repeat(4, $cardWidth);
+  grid-column-gap: 48px;
+
+  margin: 0 auto;
+
+  @media (max-width: 1024px) {
+    grid-template-columns: repeat(3, $cardWidth);
+  }
```
Comment (illiarerun) [inline src/styles/blocks/_catalog.scss:11]: and here

**Review by illiarerun — CHANGES_REQUESTED** [review]

File: src/styles/blocks/_catalog.scss [hunk 7]
```diff
@@ -0,0 +1,31 @@
...
+  @media (max-width: 1023px) {
+    grid-template-columns: repeat(3, $cardWidth);
+  }
+
+  @media (max-width: 767px) {
+    grid-template-columns: repeat(2, $cardWidth);
+  }
+
+  @media (max-width: 487px) {
+    grid-template-columns: repeat(1, $cardWidth);
```
Comment (illiarerun) [inline src/styles/blocks/_catalog.scss:17]: no neec to repeat one column
```suggestion
    grid-template-columns: $cardWidth;
```

**Review by illiarerun — APPROVED** [review]

**Pull request closed without merging**